<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QWidget" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="monitorLayout">
    <item>
     <layout class="QHBoxLayout" name="monitorDurationLayout">
      <item>
       <widget class="QLabel" name="txtlabel_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="durationComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxCount">
         <number>2147483643</number>
        </property>
        <item>
         <property name="text">
          <string>1 Hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 Hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 Hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Day</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Month</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Year</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="graphDisplayLayout">
      <item>
       <layout class="QHBoxLayout" name="graphTypeLayout">
        <item>
         <widget class="QLabel" name="txtlabel_1">
          <property name="text">
           <string>Select graph:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="graphTypeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Bytes Received</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bytes Sent</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Connection Age</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Status</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Xmpp Time</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="graphImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>graph image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
